lib/logstash/config/config_ast.rb

```diff
@@ -155,7 +155,9 @@ def compile_initializer
 
               @logger.debug? && @logger.debug(\"Flushing\", :plugin => #{name}, :events => events)
 
-              #{plugin.compile_starting_here.gsub(/^/, "  ")}
+              events.each do |event|
+                #{plugin.compile_starting_here.gsub(/^/, "  ")}
```

Comment:
not sure what this is supposed to do: expand `event.size` times the `plugin.compile_starting_here.gsub(/^/, "  ")`